bgpd/bgp_network.c

```diff
@@ -657,6 +657,11 @@ static int bgp_update_address(struct interface *ifp, const union sockunion *dst,
 		return 1;
 
 	prefix2sockunion(sel, addr);
+
+	if (IN6_IS_ADDR_LINKLOCAL(&addr->sin6.sin6_addr)) {
```

Comment:
no braces for single line if statements

Reply:
I have removed this code as it is part of another PR (https://github.com/FRRouting/frr/pull/7434).